ext/js/language/language-transformer.js

```diff
@@ -147,6 +147,19 @@ export class LanguageTransformer {
         return results;
     }
 
+    /**
+     * @param {string[]} inflectionRules
+     * @returns {import('dictionary').InflectionRuleChain}
+     */
+    addInflectionRulesDescriptions(inflectionRules) {
+        return inflectionRules.map((rule) => {
+            const fullRule = this._transforms.find((transform) => transform.name === rule);
```

Comment:
To avoid lugging around an object with the name and description, I've left the translator to work with just the array of rule names, then added finding the description here. However, if multiple rules have the same name, this finds the description for the first one. I think only `kansai-ben` is currently used as a name for multiple rules, so not what's best here.

- enforce unique rule names
- complicate the translator code by lugging around the description
- add another attribute to the rules to separate ID and display name
- use the index instead of the name internally, then find name and description here (might complicate debugging)
- smth else?

Reply:
I think this is fine for now. I think enforcing unique rule names could be a future endeavor

Reply:
I wasn't exactly clear about the problem - the kansai-ben inflection rules get the wrong toast shown, since it always shows the description for the first rule named `kansai-ben` and there are multiple. It's not that big of an issue since these rules aren't common, but still.

I'm leaning towards changing the transforms to an object:
![image](https://github.com/themoeway/yomitan/assets/24891609/27e4d3d9-47fe-456f-8981-e08358842f78)
so the keys will be unique and we can use them in the code, no ambiguity, while still allowing the user-facing `name` to be the same between a family of rules.